frontend/src/container/OnboardingContainer/Steps/ConnectionStatus/ConnectionStatus.tsx

```diff
@@ -57,8 +64,64 @@ export default function ConnectionStatus(): JSX.Element {
 		maxTime,
 		selectedTime,
 		selectedTags,
+		options: {
+			enabled: getStartedSource !== 'kafka',
+		},
+	});
+
+	const [pollInterval, setPollInterval] = useState<number | false>(10000);
+	const {
+		data: onbData,
+		error: onbErr,
+		isFetching: onbFetching,
+	} = useOnboardingStatus({
+		enabled: getStartedSource === 'kafka',
+		refetchInterval: pollInterval,
 	});
 
+	const [
+		shouldRetryOnboardingCall,
+		setShouldRetryOnboardingCall,
+	] = useState<boolean>(false);
+
+	useEffect(() => {
+		if (getStartedSource === 'kafka') {
```

Comment:
even undefined / empty string is not equal to `kafka` , not really helpful in code understanding

Reply:
added comment - for better understanding